website/content/docs/import/index.mdx

```diff
@@ -134,20 +134,56 @@ mapping_regex {
 - `destination` `(string: <required>)` - The name of a previously defined
   destination block **to** which the data should be written.
 
-- `priority` `(integer: <required>)` - The order in which Vault should apply the
-  mapping block during the import process. The lower the number, the higher the
-  priority. For example, a mapping with priority 1 executes before a mapping
-  with priority 2.
+- `filter` `(string: "")` - Optional string containing a https://github.com/hashicorp/go-bexpr
+  expression to limit which secrets from the source will be imported.
+
+#### Filters
+
+A filter describes conditions that a source secret must meet in order to be imported.
+These conditions are applied to the secret fields, which are:
+
+- `Secret.Name` - The name of the source secret
+- `Secret.Tags` - A map of key-value user-defined metadata associated with the source secret
+
+Simple binary conditions can be applied to input values:
 
-Depending on the filter type, additional fields may be required or possible. Refer
-to the [import mappings documentation](/vault/docs/import/mappings) for the available
-supported options and for a list of each mapping's specific fields.
+- `Secret.Name != "foo"`
+- `Secret.Tags.Team matches "my-dept.*"`
+- `Secret.Tags contains "my-required-tag-name"`
 
-<Tip title="Priority matters">
+These can be combined using `and`, `or`, and `not`:
 
-  Vault applies mapping definitions in priority order and a given secret only
-  matches to the first mapping that applies. Once Vault imports a secret with a
-  particular mapping, subsequent reads from the same source will ignore that
-  secret. See the [priority section](/vault/docs/import/mappings#priority) for an example.
+- `(Secret.Name == "foo" or Secret.Tags.Team == "my-team") and Secret.Tags.Category = "my-cat"`
 
-</Tip>
+#### Transformers
+
+Transform stanzas come in two forms: `exact` and `regexp`.
+
+An exact transform allows renaming a secret during import, so that the `from` secret name
+is imported into Vault as a secret named `to`.  In the example below, a source secret named
+`foo` will be imported as `foosball`.
```

Comment:
```suggestion
`exact` transforms rename secrets during import so the `from` secret name becomes the `to` secret name. In the following example , takes a source secret named
`foo` and transforms it to `foosball` during import.
```
Style correction: write in the active voice

Reply:
I think "In the following example , takes a source secret named `foo` and transforms it to `foosball` during import" doesn't sound quite right, there seems to be a missing subject.  I'll try it with "it" as the subject.

Reply:
Attempted rewrite in 6f7c69cb1a43583263b6691ee1631c8e03ef9575